### PR TITLE
Version upgrade to 6.0.0-m1-SNAPSHOT

### PR DESCRIPTION
**Pull request by minolih**
$subject

### CODE REVIEW

File: modules/api-resources/api-resources-full/pom.xml.versionsBackup [hunk 1]
```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ashensw) [inline modules/api-resources/api-resources-full/pom.xml.versionsBackup:1]: We need to remove this file.

File: modules/styles/product/pom.xml.versionsBackup [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (piraveena) [inline modules/styles/product/pom.xml.versionsBackup:2]: add `pom.xml.versionsBackup` into gitignore file and we should not commit it

File: modules/api-resources/api-resources-full/pom.xml.versionsBackup (hunk 1 above)
Comment (minolih) [reply modules/api-resources/api-resources-full/pom.xml.versionsBackup]: I have added it to git ignore. Updating the changes now.

File: modules/styles/product/pom.xml.versionsBackup (hunk 2 above)
Comment (minolih) [reply modules/styles/product/pom.xml.versionsBackup]: Done. Will make changes.

**Comment by minolih** [comment]
The PR automatically closed when head was reset. 
Changes in new PR https://github.com/wso2/product-is/pull/13704.

File: modules/api-resources/api-resources-full/pom.xml.versionsBackup (hunk 1 above)
Comment (minolih) [reply modules/api-resources/api-resources-full/pom.xml.versionsBackup]: PR automatically closed with head reset. Changes in new PR.

File: modules/styles/product/pom.xml.versionsBackup (hunk 2 above)
Comment (minolih) [reply modules/styles/product/pom.xml.versionsBackup]: PR automatically closed with head reset. Changes in new PR https://github.com/wso2/product-is/pull/13704.

**Pull request closed without merging**